src/client/index.tsx

```diff
@@ -1,8 +1,14 @@
 import * as React from 'react';
 import { render } from 'react-dom';
-import App from './routes/Routes';
+import Routes from './routes/Routes';
 import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
```

Comment:
Provider is not required  here

please add the rule for unused object

Reply:
Working on the ts-lint issue. can you suggest where I will get the all the required linting configuration?